p0_user_stories.md

```diff
@@ -6,13 +6,13 @@ Taken from https://docs.google.com/document/d/10t4q5XO1ED2PnK3ishn4y3G4Tma7uMYge
 
 ## tier-1
 
-- I want 2 apps in different namespaces to connect, but can't read (or write) the labels for those namespaces from my Kubernetes client.   I cannot select pods for this because I don't know all the labels and names of pod/services that i want to contact ahead of time. 
+- I want 2 apps in different namespaces to connect, but can't read (or write) the labels for those namespaces from my Kubernetes client.   I cannot select pods for this because I don't know all the labels and names of pod/services that I want to contact ahead of time. 
 
-- I want different pods (with potentially different labels) that fulfill a service to be able talk to each other, without talking to other services...  but don't know the labels on these services.
+- I want different pods (with potentially different labels) that fulfill a service to be able talk to each other, without talking to other services but I don't know the labels on these services.
 
-- I don't want to directly update my CIDR rules for a policy every time I add a new node or other group of IPs, which need to have policy's associated with them.
+- I don't want to directly update my CIDR rules for a policy every time I add a new node or other group of IPs, which need to have policies associated with them.
```

Comment:
This is related to https://docs.google.com/document/d/1AtWQy2fNa4qXRag9cCp5_HsefD7bxKe3ea2RPn8jnSs/edit#heading=h.ajvcztp6cza and my understanding here is creating maybe an object that groups a block of CIDR and then referencing those CIDR in the Network Policy (@jayunit100 is this the idea?).

I've been thinking about this approach, I think it makes more sense in the 'Ports' object, where I want to create a PortSet containing ports 80, 443, 8443, call it HTTPPorts and use this instead of a multiport in a network policy

Reply:
#10